wls-gui-wahllokalsystem/src/stores/wahlscheineStore.ts

```diff
@@ -0,0 +1,29 @@
+import type { Wahlscheine } from "@/types/ereignismeldung/Wahlscheine.ts";
+
+import { defineStore } from "pinia";
+import { ref } from "vue";
+
+import { useHmrUpdate } from "@/composables/common/hmrUpdate.ts";
+import { useWahlscheineService } from "@/composables/ergebnismeldung/wahlscheineService.ts";
+
+const { registerStoreHMR } = useHmrUpdate();
+const { getWahlscheine } = useWahlscheineService();
+
+export const useWahlscheineStore = defineStore("wahlscheine", () => {
```

Comment:
bisher haben wir die storeID immer ausgelagert
```suggestion
export const storeID = "wahlscheine";

export const useWahlscheineStore = defineStore(storeID, () => {
```